File: 1482-HowManyNumbersAreSmallerThanTheCurrentNumber/1482-HowManyNumbersAreSmallerThanTheCurrentNumber.py
# Last updated: 7/30/2025, 12:23:15 PM
class Solution:
    def smallerNumbersThanCurrent(self, nums: List[int]) -> List[int]:
        ans = []
        for i in range(len(nums)):
            count = 0
            for j in range(len(nums)) :
                if nums[j] < nums[i] :
                    count += 1
            ans.append(count)
        return ans
            

        

File: 1482-HowManyNumbersAreSmallerThanTheCurrentNumber/1482-HowManyNumbersAreSmallerThanTheCurrentNumber.java
// Last updated: 7/30/2025, 3:39:08 PM
class Solution {
    public int[] smallerNumbersThanCurrent(int[] nums) {
        int[] ans = new int[nums.length];

        for(int i=0; i<nums.length; i++){
            ans[i] = counter(nums,nums[i]);
        }
        return ans;
    }

    public int counter(int[] nums, int n){
        int count = 0;
        for(int i=0; i<nums.length; i++){
            if(nums[i] < n) count++;
        }
        return count;
    }
}